scripts/generate_bls_proof_of_possesion

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env python3
+import argparse
```

Comment:
Please add the script to `setup.py` so that it's installed with the build

Reply:
Done